app/src/main/java/com/project/pv239/customtimealarm/database/entity/Alarm.java

```diff
@@ -36,14 +38,23 @@
     @ColumnInfo(name = "longitude")
     private double longitude;
 
-    public Alarm(String destination, String timeOfArrival, int trafficModel, int travelMode,
-                 double latitude, double longitude){
+    @ColumnInfo(name = "on")
+    private boolean on;
+
+    @ColumnInfo(name = "morning_routine")
+    private int morningRoutine;
+
+    public Alarm(String destination, int hour, int minute, int trafficModel, int travelMode,
+                 double latitude, double longitude, boolean on, int morningRoutine){
```

Comment:
Tu je asi zly ten konstruktor...ked si predstavim flow, tak...vytvoris si novy alarm...ale hour a minute mu do konstruktoru nevies poslat ten sa dorata a nastavi dodatocne, podla mna to tam je zbytocne...just saying :)

Reply:
tak musi tam byt nejaka defaut hodnota asi ajtak cize to je asi jedno

Reply:
ahh okej moze byt :D